<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Водичка</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>330</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>11</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Объект:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Удалить объект</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Открыть файл</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="1" colspan="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>         Интенсивность:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="z_light_input">
           <property name="minimum">
            <double>-50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="x_light_input"/>
         </item>
         <item row="5" column="2">
          <widget class="QDoubleSpinBox" name="int_light_input">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="y_light_input"/>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>         Источник света</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="1" column="0">
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="simulateBtn">
             <property name="text">
              <string>Запуск симуляции</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="animateBtn">
             <property name="text">
              <string>Анимация</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Отрисовать</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>render()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1094</x>
     <y>897</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>changeLight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1026</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>1224</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>957</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>1226</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simulateBtn</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>simulate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animateBtn</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>animate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>render()</slot>
  <slot>remove()</slot>
  <slot>changeLight()</slot>
  <slot>add()</slot>
  <slot>rotate()</slot>
  <slot>move()</slot>
  <slot>load()</slot>
  <slot>simulate()</slot>
  <slot>animate()</slot>
 </slots>
</ui>
